serializer/serializer.c

```diff
@@ -6,8 +6,9 @@ void fill_tag(char* new_tag, char* old_tag, size_t tag_len) {
   sprintf(new_tag, "%s%*.*s", old_tag, pad_len, pad_len, nines);
 }
 
-int hash243_stack_to_json_array(hash243_stack_t stack, cJSON* const json_root,
-                                char const* const obj_name) {
+int ta_hash243_stack_to_json_array(hash243_stack_t stack,
```

Comment:
Is this still needed? I don't see anywhere using it.

Reply:
I think it's more appropriate to removed it in another PR.